js/fieldmanager.js

```diff
@@ -114,6 +114,25 @@ var match_value = function( values, match_string ) {
 	return false;
 }
 
+/**
+ * Initializes triggers to conditionally hide or show fields.
+ */
+var init_display_if = function() {
+	$( '.display-if' ).each(function() {
```

Comment:
It seems like this could collide with another class in global scope. Can we narrow scope?